src/pages/Organize/License/index.jsx

```diff
@@ -0,0 +1,147 @@
+import { getAllLicense } from "../../../services/licenses";
+
+const AdviceLicenses = () => {
+  const entriesOptions = [
+    {
+      id: 10,
+      value: 10,
+    },
+    {
+      id: 25,
+      value: 25,
+    },
+    {
+      id: 50,
+      value: 50,
+    },
+    {
+      id: 100,
+      value: 100,
+    },
+  ];
+
+  const initialState = {
+    page: 1,
+    limit: 100,
+    groupName: "",
+  };
+  const initialMessage = {
+    type: "success",
+    text: "",
+  };
+  const [licenseData, setLicenseData] = useState(initialState);
+  const [licenseDataList, setLicenseDataList] = useState();
+  const [showMessage, setShowMessage] = useState(false);
+  const [message, setMessage] = useState(initialMessage);
+  const handleChange = (e) => {
+    setLicenseData({
+      ...licenseData,
+      [e.target.name]: e.target.value,
+    });
+  };
+  const handleSubmit = (e) => {
+    e.preventDefault();
+  };
+  useEffect(() => {
+    getAllLicense(licenseData)
+      .then((res) => {
+        setLicenseDataList(res);
+      })
+      .catch((error) => {
+        setMessage({
+          type: "danger",
+          text: error.message,
+        });
+        setShowMessage(true);
+      });
+  }, [licenseData]);
+  return (
+    <>
+      {showMessage && (
+        <Alert
+          type={message.type}
+          setShow={setShowMessage}
+          message={message.text}
+        />
+      )}
+      <div className="main-container my-3">
+        <div className="row">
+          <div className="col-lg-12 col-md-12 col-sm-12 col-12">
+            <h1 className="font-size-main-heading">Candidate Licenses</h1>
+            <br />
+            <div className="d-flex justify-content-between">
+              <InputContainer
+                name="limit"
+                type="select"
+                onChange={(e) => handleChange(e)}
+                options={entriesOptions}
+                property="value"
+              />
+              <input
+                type="search"
+                className="form-control w-25 mt-4"
+                placeholder="Search"
+              />
+            </div>
+            <table className="table table-striped text-primary-color font-size-medium table-responsive-sm table-bordered">
+              <thead>
+                <tr className="font-bold text-center font-size-sub-heading">
+                  <th>Edit</th>
+                  <th>Short Name</th>
+                  <th>Full Name</th>
+                  <th>Text</th>
+                  <th>URL</th>
+                  <th>Merge Request</th>
+                </tr>
+              </thead>
+              <tbody>
+                {licenseDataList &&
+                  licenseDataList.map((license) => (
+                    <tr className="text-center" key={license.id}>
+                      <td></td>
+                      <td>{license.shortName}</td>
+                      <td>{license.fullName}</td>
+                      <td>{license.text}</td>
```

Comment:
Please add text in a disabled `textarea` or any similar element for easy viewing.

Reply:
For the candidate License i don' think so we are not getting any such bix text please check once.

Reply:
Yes because that is from your local. This is how it looks for me:

![image](https://user-images.githubusercontent.com/18077542/124786285-36434000-df65-11eb-9f95-4d8e3788b08b.png)

Reply:
I guess I have updated it in the later commit in the same pr, probably we can discuss this in the meeting.

Reply:
Nope, still looks the same for me.

![image](https://user-images.githubusercontent.com/18077542/124913825-37c84300-e00d-11eb-9417-447761dfa98b.png)

Reply:
@GMishx this pr itself has 4 commits because I have added admin license pages as well into it I have checked downlading the zip of pr it is working completly fine, can you please check fetching all the commits.

Reply:
The PR is fine, I am also on the latest commit but I see you missed two points:
1. My original comment "Please add text in a disabled textarea or any similar element for easy viewing."
2. You can create/edit candidate licenses in fossology, it's not always "License by OJO.". So in the screenshot, I actually have these licenses as candidate licenses.

I hope this will resolve some confusions.

Reply:
@GMishx updated, please check now is it fine now

Reply:
Hi @Shruti3004 , unfortunately I still can not see the `textarea` either in the deployed code or there in github. Can you please check if you've pushed the code?

Reply:
@GMishx please check now does the things working for you fine now?